README.md

```diff
@@ -60,3 +60,11 @@ no responsibility whatsoever for its use by other parties, and makes no
 guarantees, expressed or implied, about its quality, reliability, or
 any other characteristic. We would appreciate acknowledgement if the
 software is used.
+
+## 3D-Support
+
+To enable this software to navigate through slices image in adequation with the navigation per frame, the files : openseedragon-movie.js, manifest.js, toolbar.js and viewer.js, have been updated.
+
+A new slider provide the navigation per slice when a layer have correct properties, like them of "Moche xy-energy-slice" in the file manifest.json, whithout breaking the functionnality of navigation per frame.
```

Comment:
Please use this corrected sentence instead: A new slider provides the navigation per slice when a layer has zslice properties (see example of such properties from "Moche xy-energy-slice" in the file manifest.json), without breaking the functionality of navigation per frame.